packages/opentelemetry-basic-tracer/src/platform/node/timer-util.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * When called, the active Timeout object will not require the Node.js event
+ * loop to remain active.
+ */
+export function unrefTimer(timer: NodeJS.Timeout): void {
+  timer.unref();
```

Comment:
How likely is it that we will need to unref timers in other places in the code base? Would it make sense to put this in the `core` package?

Reply:
(Well, other places in the code base that we want to make shared with the browser that is)

Reply:
Not sure for now, I think will consider to move in core in the future (if needed).

Reply:
OK, makes sense.